resolwe/flow/models.py

```diff
@@ -273,22 +272,10 @@ class Meta(BaseModel.Meta):
 
 
 def render_template(template_string, context):
-    """Render template based on Dango template language."""
-    template_headers = [
-        '{% load resource_filters %}',
-        '{% load process_fields %}',
-        '{% load mathfilters %}',
-    ]
-
-    custom_template_tags = getattr(settings, 'RESOLWE_CUSTOM_TEMPLATE_TAGS', [])
-    if not isinstance(custom_template_tags, list):
-        raise KeyError("`RESOLWE_CUSTOM_TEMPLATE_TAGS` setting must be a list.")
-
-    template_headers.extend(
-        ['{{% load {} %}}'.format(template_tag) for template_tag in custom_template_tags]
-    )
+    """Render template using the Jinja expression engine."""
+    from .managers import manager
 
-    return template.Template(''.join(template_headers) + template_string).render(context)
+    return manager.get_expression_engine('jinja').evaluate_block(template_string, context)
```

Comment:
Well, not sure how this would be determined otherwise? Previously this always used DTL. We currently don't have any process-wise definition of which expression engine should be used for fields like `data_name`. In CWL, there are the `requirements` that a process can define, but we don't have that (only a language for the program, but this is something different).

One possible way would be to add something like the `requirements` from CWL, so the process definition would have to say something like this at the top level:
```yaml
requirements:
    - expression-engine:jinja
```

And then everything would use that expression engine. We could also then just have the `bash` execution engine instead of `bash-jinja` and reuse this configuration also for expressions inside the program. This would also bring our implementation closer to CWL. What do you think?

/cc @mstajdohar @tjanez